docs/configuration/executor/index.md

```diff
@@ -53,6 +53,27 @@ executors:
 
 For all executor configuration properties, see the [**syntax**](#syntax) section.
 
+#### Restart Policy and Health Check Interval
+
+This section of the configuration allows you to configure the restart policy for the Botkube executor plugins. The restart policy is used when the executor plugin fails to start. The default restart policy is `DeactivatePlugin`, which means that the plugin is deactivated after a given number of restarts. The restart policy can be configured with the following properties:
+
+- `type` - restart policy type. Allowed values: `RestartAgent`, `DeactivatePlugin`.
```

Comment:
Can we describe the possible values? One sentence per the value would be enough. 
Please apply that to each document.
Thanks!